src/pr-info.ts

```diff
@@ -422,16 +426,17 @@ configSuspicious["tsconfig.json"] = makeJsonCheckerFromCore(
         }
     },
     [ "/files", "/compilerOptions/paths", "/compilerOptions/baseUrl", "/compilerOptions/typeRoots" ],
+    JSON.parse,
     urls.tsconfigJson
 );
 
 // helper for json file testers: allow either a given "requiredForm", or any edits that get closer
 // to it, ignoring some keys (JSON Patch paths).  The ignored properties are in most cases checked
 // elsewhere (dtslint), and in some cases they are irrelevant.
-function makeJsonCheckerFromCore(requiredForm: any, ignoredKeys: string[], requiredFormUrl: string) {
+function makeJsonCheckerFromCore(requiredForm: any, ignoredKeys: string[], parse: (text: string) => unknown, requiredFormUrl: string) {
     const diffFromReq = (text: string) => {
         let json: any;
-        try { json = JSON.parse(text); } catch (e) { return "couldn't parse json"; }
+        try { json = parse(text); } catch (e) { return "couldn't parse json"; }
```

Comment:
Kind of picky since the line-splitting "parser" is unlikely to fail, but still, maybe:

```suggestion
        try { json = parse(text); } catch (e) { return `couldn't parse${parse === JSON.parse ? " json" : ""}`; }
```